src/Worker.Extensions.DurableTask/Worker.Extensions.DurableTask.csproj

```diff
@@ -39,12 +39,12 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.5.0-preview1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.6.0-preview1" />
```

Comment:
FYI: preview3 is the [latest](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Core/1.6.0-preview3)

Reply:
Oh, looks like I missed that! Any interesting improvements I might have missed in those two releases?

Reply:
No impact to the durable use cases. It was minor fixes in other areas.